src/main/resources/tags/info/advertising.tag

```diff
@@ -0,0 +1,10 @@
+type: text
+aliases: advert, advertise
+
+---
+
+We no longer allow advretising in <#754458074818805811>, however, there are two other option available to you:
```

Comment:
```suggestion
We no longer allow advertising in <#754458074818805811>, however, there are two other option available to you:
```

Reply:
ah yes. i though i checked over it all... oops. For some reason the spelling check built into my browser doesn't work in those data entry boxes on github